manifest.php

```diff
@@ -135,4 +136,7 @@
     'extra' => [
         'structures' => __DIR__ . DIRECTORY_SEPARATOR . 'actions' . DIRECTORY_SEPARATOR . 'structures.xml',
     ],
+    'containerServiceProviders' => [
+        ItemsReferencesServiceProvider::class
```

Comment:
I think you also need to increase the min version in composer.json to use this feature:

```
"oat-sa/generis" : ">=15.5.0",
"oat-sa/tao-core" : ">=48.39.0",
```